stripe/api_resources/recipient_transfer.py

```diff
@@ -1,6 +0,0 @@
-from stripe.stripe_object import StripeObject
-
-
-# This resource can only be instantiated when expanded on a BalanceTransaction
-class RecipientTransfer(StripeObject):
```

Comment:
Where did this file go? I can't seem to find it if it was renamed

Reply:
Nice catch, this class is unused and was lost in all the moves.